app/src/main/res/values/strings.xml

```diff
@@ -116,11 +116,14 @@
 
     <!-- Comments -->
     <string name="content_description_comment_author_avatar">Author avatar</string>
+    <string name="content_description_downvote_arrow">Down-vote Arrow</string>
+    <string name="content_description_upvote_arrow">Up-vote Arrow</string>
     <string name="no_comments_yet">No comments yet. Be the first to post a comment</string>
     <string name="reply">Reply</string>
     <string name="reply_to_">Reply to %s</string>
     <string name="cancel_reply">Cancel reply</string>
     <string name="post">Post</string>
     <string name="add_comment_success">Comment added successfully</string>
+    <string name="vote_placeholder">0</string>
```

Comment:
Empty would be better, 0 does not add much value

Reply:
I've removed this line and left the text attribute empty on the button.